isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -24,30 +24,89 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
 -->
   <h2>Overview</h2>
 
+  <h3>ISIS3 Installation With Conda</h3>
+    <p>The instructions for this are still being worked out.</p>
+
   <h3>Operating System Requirements</h3>
-  <p>ISIS runs on many UNIX variants. ISIS does not run on MS Windows. The UNIX variants supported are listed here:</p>
+  <p>ISIS3 runs on many UNIX variants. ISIS does not run natively on MS Windows,
+     although it has been successfully run on Windows 10 using the Windows
+     Subsystem for Linux (WSL).  Instructions for doing this can be found
+     <a href="#RunningOnWindows">here.</a>
+
+    The UNIX variants ISIS3 has been successfully built on are:</p>
     <ul>
       <li>Ubuntu 18.04 LTS</li>
       <li>Mac OS X 10.13.6 High Sierra</li>
       <li>CentOS Linux 7.2.1511</li>
       <li>Fedora 28</li>
     </ul>
 
+
+    <p>ISIS3 is also build and tested nightly on the following operating systems.</p>
+
+    <ul>
+      <li>Ubuntu 14.04.3 LTS</li>
+      <li>Debian Linux 8 (jessie)</li>
+      <li>CentOS Linux 7.2.1511</li>
+      <li>Fedora 25</li>
+      <li>Mac OS X 10.11.6</li>
+    </ul>
+<p>
+    ISIS3 maybe run on other operating systems then those listed above,
+    but it has not been tested.
+</p>
    <h3>Hardware Requirements</h3>
    <p>Here are the minimum hardware requirements</p>
     <ul>
       <li>64-bit (x86) processors</li>
-      <li>2 GB memory</li>
-      <li>10 GB to 180 GB disk space for ISIS installation</li>
+      <li>2 GB RAM</li>
+      <li>2.5 GB for ISIS3 binaries</li>
+      <li>10 GB to 510 GB disk space for ISIS3 data</li>
       <li>10 GB to many TB disk space for processing images</li>
       <li>A quality graphics card</li>
     </ul>
-<p>Note: More processors, memory, disk storage, and an additional graphcis card (to support the use of two monitors at
-one time) may be useful depending on the complexity of your processing requirements
+<p>Note: More processors, memory, disk storage, and an additional graphcis card
+  (to support the use of two monitors at one time) may be useful depending on
+  the complexity of your processing requirements.
 </p>
 
+<A NAME="RunningOnWindows"> </A>
+<h3>Running ISIS3 on Windows 10</h3>
+  <ul>
+    <li><a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10">Installing the Windows Subsystem for Linux</a></li>
+    <li><a href="http://planetarygis.blogspot.com/2017/07/isis3-on-windows-10-bash.html">Instructions for using the Windows
+      Subsystem for Linux (WSL) to run ISIS3</a></li>
+  </ul>
+  <p>
+  While the ISIS3 development team has not examined these
+  instructions for correctness or completeness, they appear to have
+  successfully worked for a number of ISIS3 users within the USGS who run Windows.
```

Comment:
I think we are beyond 'they appear'. Why not just say, 'please test and open an issue' or something to that extent. We are all one team in the building.

Reply:
Are we going to be supporting Windows? Is that a part of the ISIS support/maintenance deliverables this year?

Because if we aren't going to support it and we tell them to open an issue they are going to get ignored.